id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/service/impl/IdRepoProxyServiceImpl.java

```diff
@@ -591,6 +588,10 @@ private void getBiometricFiles(Uin uinObject, List<DocumentsDTO> documents, Map<
 				} catch (IOException e) {
 					mosipLogger.error(IdRepoSecurityManager.getUser(), ID_REPO_SERVICE_IMPL, GET_FILES, e.getMessage());
 					throw new IdRepoAppUncheckedException(FILE_STORAGE_ACCESS_ERROR, e);
+				} catch (ObjectStoreAdapterException e) {
+					mosipLogger.error(IdRepoSecurityManager.getUser(), ID_REPO_SERVICE_IMPL, "getBiometricFiles",
+							"FILE NOT FOUND IN OBJECT STORE");
+					throw new IdRepoAppUncheckedException(FILE_NOT_FOUND);

```

Comment:
we should throw in similar pattern. like above

throw new IdRepoAppUncheckedException(e.getErrorCode(), e.getErrorText(), e);